trace/obfuscation_config.yaml

```diff
@@ -0,0 +1,9 @@
+sensitive_keys:
+  - x-access-token
+  - encryption_key
+  - sso_jwt_key
+  - postman_sid
+  - api_key
```

Comment:
x-api-key?

Reply:
I have not added all the headers and regexes yet. I will add once the implementation part is reviewed in a seperate PR.
_Mentioned this above also as comment_

Reply:
OK, I don't think that is clear from your tickets -- I want to make sure that we don't miss that.

Be sure to get the shared secret keyword used by the support portal.